<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>熄灯游戏 - 探索模式</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetSize">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>设置大小</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setState">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>设置全体状态</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="stateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>104</width>
     <height>181</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="posEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>104</width>
     <height>151</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>影响周围坐标:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pos_yes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pos_no">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>状态转移总表:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="state_yes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="state_no">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="diytext">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义单元格状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前状态转移:</string>
   </property>
  </widget>
  <widget class="QLabel" name="curStateMap">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>101</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存为文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>从文件加载</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
